ts/IterableInboxMessageList.tsx

```diff
@@ -12,23 +12,23 @@ import {
 type MessageListProps = {
    rowViewModels: InboxRowViewModel[],
    customizations: IterableInboxCustomizations,
-   deleteRow: Function,
    messageListItemLayout: Function,
+   deleteRow: Function,
    handleMessageSelect: Function,
    contentWidth: number,
    orientation: string 
 }
 
 const IterableInboxMessageList = ({ 
    rowViewModels,
-   customizations, 
+   customizations,
+   messageListItemLayout, 
    deleteRow, 
-   messageListItemLayout,
    handleMessageSelect,
    contentWidth,
    orientation
 }: MessageListProps) => {
-   const [swiping, setSwiping] = useState(false)
+   //const [swiping, setSwiping] = useState(false)
```

Comment:
If we don't think it'll have relevance any time soon (and I don't think it has for a while), we should just delete it.

Reply:
Yes, I will remove the swiping flag. This will be addressed in PR-219 => [https://github.com/Iterable/react-native-sdk/pull/219](url)